plugins/scanners/scanoss/src/test/kotlin/ScanOssResultParserTest.kt

```diff
@@ -208,5 +208,19 @@ class ScanOssResultParserTest : WordSpec({
                     TextLocation("kernel/hung_task.c", 86, 107)
             }
         }
+
+        "should exclude identified snippets from snippet findings" {
```

Comment:
"should" should not be part of the test name, as the surrounding container already uses the `should` keyword`.